API/testrunner/testrunner.py

```diff
@@ -43,6 +43,7 @@ def __update_status_icon(self):
         if not utils.exists(results_web_path):
             return
 
+        utils.execute(results_web_path, 'git', ['fetch'])
```

Comment:
I think the `fetch` is not enough because it doesn't update the local gh-pages branch. (It just fetches under an `origin/gh-pages` branch.)

What about to use `utils.execute(results_web_path, 'git', ['pull', 'origin', 'gh-pages'])`?

Reply:
@rtakacs : You're right! However it's working already, and it updates the remote-tracking branches, it's safer to bring also the local branch up-to-date with its remote version. I've updated this patch accordingly.